crates/nargo_cli/tests/execution_success/references/src/main.nr

```diff
@@ -1,4 +1,6 @@
 fn main(mut x: Field) {
+    regression_2218(x, 10);
```

Comment:
Can you move this to the end with the other regressions?

Reply:
done